egs/wsj/s5/steps/nnet3/train_raw_dnn.py

```diff
@@ -198,6 +198,10 @@ def train(args, run_opts):
     try:
         model_left_context = variables['model_left_context']
         model_right_context = variables['model_right_context']
+        if 'add_lda' in variables:
+            add_lda = common_lib.str_to_bool(variables['add_lda'])
```

Comment:
I'll merge anyway because this isn't related to the merge conflict, but I'm a little surprised you are using the 'add_lda' variable; that was removed, IIRC, and we are using the presence of 'init.config' (or not) as the signal for whether or not to compute the preconditioning matrix.  Why do you need this variable?